app/scripts/move-popup/dimMoveItemProperties.directive.js

```diff
@@ -152,5 +153,50 @@
         });
       }
     }
+
+    /*
+    * Check that the current item is infusable
+    * Only legendary or Exotic items are infusable.
+    */
+    vm.isInfusable = function isInfusable(item) {
+      // Infuse perk's id is 1270552711
+      return _.contains(item.talentPerks, 1270552711);
+    }
+
+    /*
+    * Show the icon for infusion, weapon or armor
+    */
+    vm.icon = function icon(item) {
+      var icon = "";
+      if (item.sort && item.sort === "Weapons") {
```

Comment:
Yup yup, fair enough! Again nice work on this! Can't wait for people to use it! So handy!

Wish we had a way to teach existing users the functionality without their own experimentation. 

Any one have any thoughts? Maybe one of the blue 'click to dismiss' toasters?